<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogControl</class>
 <widget class="QDialog" name="DialogControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>384</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>控制</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>381</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>普通</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>109</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="heartbeat_enable">
        <property name="text">
         <string>心跳信号</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>周期(ms):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="heartbeat_interval">
          <property name="text">
           <string>500</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="setMode_halt">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>设为停止模式</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setMode_lockAttitude">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>设为锁定姿态模式</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setMode_lockThrottle">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>设为锁定油门模式</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="lockThrottle">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <attribute name="title">
     <string>锁定油门</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>201</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>油门</string>
     </property>
     <widget class="QSlider" name="lockThrottle_slider0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>31</width>
        <height>81</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lockThrottle_text0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
     <widget class="QSlider" name="lockThrottle_slider1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>31</width>
        <height>81</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lockThrottle_text1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
     <widget class="QSlider" name="lockThrottle_slider2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>31</width>
        <height>81</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lockThrottle_text2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>100</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
     <widget class="QSlider" name="lockThrottle_slider3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>31</width>
        <height>81</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="pageStep">
       <number>5</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lockThrottle_text3">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="lockThrottle_same">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>统一</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="lockAttitude">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <attribute name="title">
     <string>锁定姿态</string>
    </attribute>
    <widget class="ControlLockAttitudeWidget" name="lockAttitude_widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>301</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>51</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string>油门</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>51</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QSlider" name="lockAttitude_throttleBase">
         <property name="minimumSize">
          <size>
           <width>45</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::WheelFocus</enum>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lockAttitude_throttleValue">
         <property name="text">
          <string>0</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>313</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>灵敏度</string>
     </property>
     <widget class="QSlider" name="lockAttitude_hsSensitivity">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="value">
       <number>49</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lockAttitude_lSensitivity">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>15</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>弧度/像素</string>
      </property>
      <property name="text">
       <string>0.005</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="lockAttitude_btReset">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>复位</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btGetReady">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>准备起飞</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="lockAttitude_cbCameraControl">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>摄像头控制</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lockAttitude_btCameraSetting">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>摄像头设置</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ControlLockAttitudeWidget</class>
   <extends>QWidget</extends>
   <header location="global">dialogcontrol/controllockattitudewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>setMode_lockAttitude</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>setMode_halt</tabstop>
  <tabstop>setMode_lockThrottle</tabstop>
  <tabstop>heartbeat_interval</tabstop>
  <tabstop>heartbeat_enable</tabstop>
  <tabstop>lockThrottle_slider0</tabstop>
  <tabstop>lockThrottle_slider1</tabstop>
  <tabstop>lockThrottle_slider2</tabstop>
  <tabstop>lockThrottle_slider3</tabstop>
  <tabstop>lockThrottle_same</tabstop>
  <tabstop>btGetReady</tabstop>
  <tabstop>lockAttitude_throttleBase</tabstop>
  <tabstop>lockAttitude_btReset</tabstop>
  <tabstop>lockAttitude_hsSensitivity</tabstop>
  <tabstop>lockAttitude_cbCameraControl</tabstop>
  <tabstop>lockAttitude_btCameraSetting</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
